From: Michael Fladischer <FladischerMichael@fladi.at>
Date: Sat, 19 Dec 2015 21:54:38 +0100
Subject: Fix spelling error in multiprocessing.c.

---
 Modules/_billiard/multiprocessing.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/_billiard/multiprocessing.c b/Modules/_billiard/multiprocessing.c
index 0d5af9d..ad9a410 100644
--- a/Modules/_billiard/multiprocessing.c
+++ b/Modules/_billiard/multiprocessing.c
@@ -323,7 +323,7 @@ bail:
 static PyMethodDef Billiard_module_methods[] = {
     {"address_of_buffer", Billiard_multiprocessing_address_of_buffer, METH_O,
      "address_of_buffer(obj) -> int\n\n"
-     "Return address of obj assuming obj supports buffer inteface"},
+     "Return address of obj assuming obj supports buffer interface"},
 #if HAVE_FD_TRANSFER
     {"sendfd", Billiard_multiprocessing_sendfd, METH_VARARGS,
      "sendfd(sockfd, fd) -> None\n\n"
